src/components/FormAlertWithSubmitButton.tsx

```diff
@@ -82,6 +84,10 @@ function FormAlertWithSubmitButton({
     const style = [!footerContent ? {} : styles.mb3, buttonStyles];
     const safePaddingBottomStyle = useSafePaddingBottomStyle();
 
+    // Disable pressOnEnter only for Android Native due to bug here: https://github.com/Expensify/App/issues/46644
```

Comment:
Can we explain the bug in comment please. i mean a specific reason to disable enter

Reply:
@nyomanjyotisa ^